.pre-commit-config.yaml

```diff
@@ -68,3 +68,9 @@ repos:
             )$
         additional_dependencies:
           - tomli
+
+  - repo: https://github.com/tweag/FawltyDeps
```

Comment:
We don't need to add this hook to every single commit that a user makes. I think we can run this once if there are missing dependencies that haven't been declared that we don't know about!